x-pack/plugins/watcher/common/constants/action_modes.ts

```diff
@@ -4,8 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const ACTION_MODES = {
-
+export const ACTION_MODES: { [key: string]: string } = {
```

Comment:
Typescript string-based enums (https://www.typescriptlang.org/docs/handbook/enums.html) are normally used for these list of constant values.